src/styles/blocks/container.scss

```diff
@@ -0,0 +1,38 @@
+@import "./mixin.scss";
+@import "./variables.scss";
+
+.container {
+  display: flex;
```

Comment:
Do you need this `flex `rule here? You used `position `rule to place you elements inside so it's unnecessary)